src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/TagBuilder.cs

```diff
@@ -14,11 +14,19 @@
 
 namespace Microsoft.AspNet.Mvc.Rendering
 {
+    /// <summary>
+    /// Contains methods and properties that are used to create HTML elements. This class is used to write HTML
+    /// helpers and Razor tag helpers.
```

Comment:
Suggest ~~that are~~ and `This class is often used in the implementation of HTML helper and tag helpers.` Not sure about including "Razor" -- see mostly just "tag helpers" elsewhere.